pp_ctl.c

```diff
@@ -4439,7 +4490,11 @@ S_require_file(pTHX_ SV *sv)
 
 PP(pp_require)
 {
-    RUN_PP_CATCHABLY(Perl_pp_require);
+    /* If a suitable JMPENV catch frame isn't present, add it and a RUNOPS
+     * loop by recursively calling ourselves, The main body of this
```

Comment:
Shouldn't this say "add it and a RUNOPS loop _then_ recursively call ourselves", here and below?